CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# CMakeLists.txt
+
+# Part of MR Lab Control Software
+
+# SPDX-License-Identifier: EUPL-1.2
+# SPDX-FileCopyrightText: Copyright (c) 2025 Martin Rumori/sonible GmbH
+
+cmake_minimum_required (VERSION 3.16)
+
+# This setting has to occur BEFORE the project call to work correctly
+set (CMAKE_OSX_DEPLOYMENT_TARGET "10.14" CACHE STRING "Minimum OS X deployment version" FORCE)
+
+project (mrlab VERSION 0.0.1)
+
+# CPM dependency management: https://github.com/cpm-cmake/CPM.cmake
+include (cmake/CPM.cmake)
+
+# Compile commands, useful for some IDEs like VS-Code
+set (CMAKE_EXPORT_COMPILE_COMMANDS TRUE)
+
+# Static linking in Windows
+set (CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
+
+# JUCE dependency
+CPMAddPackage ("gh:juce-framework/JUCE#8.0.9")
+
+#====================================================================================
+juce_add_gui_app (mrlabctrl
+    PRODUCT_NAME  mrlabctrl
+    COMPANY_NAME  sonible
+    BUNDLE_ID     com.sonible.mrlabctrl
+
+    HARDENED_RUNTIME_ENABLED TRUE
+)
+
+set_target_properties (mrlabctrl
+    PROPERTIES
+    CXX_STANDARD 20
+)
```

Comment:
For a brand new project, I'd go for the more modern approach by setting the language standard as compile feature:

```
target_compile_features (mrlabctrl PUBLIC cxx_std_20)
```